storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -37,6 +45,9 @@
         "EFM32GG11_STK3701": {
             "QSPI_MIN_READ_SIZE": "4",
             "QSPI_MIN_PROG_SIZE": "256"
+        },
+        "DISCO_F746NG": {
```

Comment:
Hi
I think issue is not really specific to the DISCO F746 board,
but more to the QSPI?
Please replace "DISCO_F746NG" by "N25Q128A"
Thx

Reply:
Very good catch! I didn't know `N25Q128A` had its own config, will change it.

Reply:
Done